server/src/test/scala/services/liveviews/PassengersLiveViewTest.scala

```diff
@@ -0,0 +1,65 @@
+package services.liveviews
+
+import drt.shared.CrunchApi.{MinutesContainer, PassengersMinute}
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import uk.gov.homeoffice.drt.db.PassengersHourlyRow
+import uk.gov.homeoffice.drt.ports.PortCode
+import uk.gov.homeoffice.drt.ports.Queues.{EGate, EeaDesk}
+import uk.gov.homeoffice.drt.ports.Terminals.T1
+import uk.gov.homeoffice.drt.time.SDate
+
+import java.sql.Timestamp
+
+class PassengersLiveViewTest extends AnyWordSpec with Matchers {
+  "minutesContainerToHourlyRow given a container with one minute" should {
+    "return one row with the the passengers for that minute" in {
+      val minute1 = SDate("2023-12-21T15:00")
+
+      val container = MinutesContainer(
+        List(PassengersMinute(T1, EeaDesk, minute1.millisSinceEpoch, Seq(0.5), None))
+      )
+
+      val code = PortCode("LHR")
```

Comment:
if code is rename to portCode it would be easy to ready

Reply:
> if code is rename to portCode it would be easy to ready

:+1: